core/drivers/crypto/versal/crypto.mk

```diff
@@ -0,0 +1,10 @@
+ifeq ($(CFG_VERSAL_CRYPTO_DRIVER),y)
+# Enable the crypto driver
+$(call force,CFG_CRYPTO_DRIVER,y)
+
+CFG_CRYPTO_DRIVER_DEBUG ?= 0
+$(call force,CFG_CRYPTO_DRV_ACIPHER,y)
+$(call force,CFG_CRYPTO_DRV_ECC,y)
+$(call force,CFG_CRYPTO_DRV_RSA,y)
+$(call force,CFG_CRYPTO_DRV_AUTHENC,y)
```

Comment:
Why `force` and not `?=`

Reply:
So the way I thought about this is that when Versal sets CFG_VERSAL_CRYPTO_DRIVER=y, the three drivers are mandatory: ie versal ecc, versal rsa and versal auth-enc. 

Those however can only be enabled if the crypto API is also configured to allow calling them (hence the forcing)